pom.xml

```diff
@@ -6,22 +6,22 @@
 
     <groupId>com.getindata</groupId>
     <artifactId>kafka-connect-iceberg-sink</artifactId>
-    <version>0.1.3-SNAPSHOT</version>
+    <version>0.1.4-SNAPSHOT</version>
```

Comment:
Version is updated autmatically during preapre release action:
https://github.com/getindata/kafka-connect-iceberg-sink/blob/36523e9b7c72529b921dbccb7fc8e4504b9bca22/.github/workflows/prepare-release.yml#L28

Reply:
changed in the new commit